sqlserver/metadata.csv

```diff
@@ -1,11 +1,24 @@
 sqlserver.access.page_splits,gauge,,operation,second,The number of page splits per second.,-1,sql_server,page splits,
+sqlserver.ao.is_primary_replica,gauge,,,,"Returns 1 if the replica is primary, or 0 if it is a secondary replica.",0,sql_server,is primary replica,
 sqlserver.ao.ag_sync_health,gauge,,,,"Availability group synchronization health: 0 = Not healthy, 1 = Partially healthy, 2 = Healthy",0,sql_server,ag sync health,
 sqlserver.ao.replica_sync_state,gauge,,,,"Replica synchronization state: 0 = Not synchronizing, 1 = Synchronizing, 2 = Synchronized, 3 = Reverting, 4 = Initializing",0,sql_server,replica sync state,
+sqlserver.ao.log_send_queue_size,gauge,,,,"Amount of log records of the primary database that has not been sent to the secondary databases, in kilobytes (KB).",0,sql_server,log send queue size,
+sqlserver.ao.log_send_rate,gauge,,,,"Average rate at which primary replica instance sent data during last active period, in kilobytes (KB)/second.",0,sql_server,log send rate,
+sqlserver.ao.redo_queue_size,gauge,,,,"Amount of log records in the log files of the secondary replica that has not yet been redone, in kilobytes (KB).",0,sql_server,redo queue size,
+sqlserver.ao.redo_rate,gauge,,,,"Average rate at which the log records are being redone on a given secondary database, in kilobytes (KB)/second.",0,sql_server,redo rate,
+sqlserver.ao.low_water_mark_for_ghosts,gauge,,,,"A monotonically increasing number for the database indicating a low water mark used by ghost cleanup on the primary database.",0,sql_server,low water mark for ghosts,
+sqlserver.ao.filestream_send_rate,gauge,,,,"The rate at which the FILESTREAM files are shipped to the secondary replica, in kilobytes (KB)/second.",0,sql_server,filestream send rate,
+sqlserver.ao.secondary_lag_seconds,gauge,,,,"The number of seconds that the secondary replica is behind the primary replica during synchronization.",0,sql_server,secondary lag seconds,
 sqlserver.ao.replica_failover_mode,gauge,,,,"Replica failover mode: 0 = Automatic failover, 1 = Manual failover",0,sql_server,replica failover mode,
 sqlserver.ao.replica_failover_readiness,gauge,,,,"Replica failover readiness: 0 = Not ready for failover, 1 = Ready for failover",0,sql_server,replica failover readiness,
 sqlserver.ao.primary_replica_health,gauge,,,,"Recovery health of primary replica: 0 = In progress, 1 = Online. The metric is not emitted if on a secondary replica",0,sql_server,primary replica health,
 sqlserver.ao.secondary_replica_health,gauge,,,,"Recovery health of secondary replica: 0 = In progress, 1 = Online. The metric is not emitted if on a primary replica",0,sql_server,secondary replica health,
+sqlserver.ao.quorum_type,gauge,,,,"Type of quorum used by the WSFC cluster, one of: 0 = Node Majority, 1 = Node and Disk Majority, 2 = Node and File Share Majority, 3 = No Majority, 4 = Unknown Quorum, 5 = Cloud Witness",0,sql_server,ao quorum type,
```

Comment:
We should not emit the enum types as values. Those should be tags and the value should be `gauge` value 1.

This means the query should look something like:

```sql

select
    1,  -- first column is used as the metric value
    quorum_type_desc,  -- all other columns are tags
    ...
```